tests/check-legal-aid/test/local.json

```diff
@@ -0,0 +1,29 @@
+{
+  "src_folders": "tests/nightwatch/specs",
+  "output_folder": "tests/reports/nightwatch",
+  "custom_assertions_path": "tests/nightwatch/custom_assertions",
+  "custom_commands_path": "tests/nightwatch/commands",
+  "globals_path": "tests/nightwatch/globals/local.js",
+
+  "selenium": {
+    "start_process": false
+  },
+
+  "test_settings": {
+    "default": {
+      "launch_url": "http://localhost:5000",
```

Comment:
Could we remove this configuration until it's necessary, please?

Reply:
:thumbsup: